src/uiSelectController.js

```diff
@@ -43,7 +43,17 @@ uis.controller('uiSelectCtrl',
   if (ctrl.searchInput.length !== 1) {
     throw uiSelectMinErr('searchInput', "Expected 1 input.ui-select-search but got '{0}'.", ctrl.searchInput.length);
   }
-  
+
+  ctrl.headerClick = function(g) {
+	console.log("header is clicked:", g);
```

Comment:
@wesleycho : I moved the code to src files as requested. That is how it is done right?